raster-foundry/raster-foundry
# Remove landsat scenes with < 0 cloud cover

## Overview

This PR adds a migration to remove Landsat 8 scenes with negative cloud cover and
skips them in the importer.

### Checklist


## Notes

I also uncovered that for a few months ago  _lot_ of Landsat 8 scenes seem to be missing from AWS, and also that my logging message about skipping scenes isn't printing, so I wonder if some filtering is
happening before scenes get to AWS now. :man_shrugging: 

## Testing Instructions

 * `select count(1) from scenes where cloud_cover < 0 and datasource = '697a0b91-b7a8-446e-842c-97cda155554d'`
 * run migration
 * reassemble batch jar
 * run a landsat 8 import
 * count again
 * should be 0

Closes azavea/raster-foundry-platform#397
